org.quassel_irc.quasselclient.json

```diff
@@ -0,0 +1,36 @@
+{
+    "app-id": "org.quassel_irc.quasselclient",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "quasselclient",
+    "rename-icon": "quassel",
+    "rename-desktop-file": "quasselclient.desktop",
+    "finish-args": [
+        "--socket=wayland",
```

Comment:
As noted, I'd probably disable wayland support for now. XWayland is still common and upstream doesn't recommend Wayland yet.

Reply:
Wayland is now no supported if you're using "--env=QT_QPA_PLATFORM=flatpak" so feel free to ignore this.

Reply:
The telegram flatpak reverted it because qt still had issues and it forced the x backend.